test/parallel/test-net-socket-destroy-twice.js

```diff
@@ -23,7 +23,7 @@
 const common = require('../common');
 const net = require('net');
 
-const conn = net.createConnection(common.PORT);
+const conn = net.createConnection(0);
```

Comment:
Port 0 with `connect()` or `createConnection()` makes no sense, except perhaps for connections that will fail for other reasons already (in which case it should *probably* have a comment explaining that).